saos-webapp/src/main/java/pl/edu/icm/saos/webapp/DefaultExceptionHandler.java

```diff
@@ -0,0 +1,20 @@
+package pl.edu.icm.saos.webapp;
+
+import javax.persistence.EntityNotFoundException;
+
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+
+@ControllerAdvice
+public class DefaultExceptionHandler {
+	
+	
+	@ExceptionHandler(EntityNotFoundException.class)
+	public String entityNotFoundExceptionErrorHandler(EntityNotFoundException e) {
+		e.printStackTrace();
+		return "judgmentNotFound";
```

Comment:
It should be more general I think, e.g. entityNotFound. What if the exception will be thrown during getting other object types from db?